fyntex/drf_pagination_utils/styles.py

```diff
@@ -123,4 +123,99 @@ def get_paginated_response_schema(self, schema: Mapping[str, object]) -> Mapping
         :param schema: Original, non-paginated response schema.
         :return: Paginated response schema.
         """
-        return schema
+        paginated_response_schema: MutableMapping[str, Mapping[str, object]] = {}
```

Comment:
@fpinto-cdd I think these changes belong to `LinkHeaderPageNumberPagination` class.

Reply:
Now I'm using the :class:`LinkHeaderPageNumberPagination` , but I had to update the method when called from  
:class:`ObjectCountHeaderPageNumberPagination` to add its related objects